templates/applications/application.html

```diff
@@ -4,136 +4,319 @@
+<script>
+function showProgressDetail(stage) {
+  const showMoreBtn = document.querySelector(`.show-more-${stage}`);
+  const showLessBtn = document.querySelector(`.show-less-${stage}`);
+  const detail = document.querySelector(`.progress-detail-${stage}`);
+  if ( detail.classList.contains("u-hide")) {
+    detail.classList.remove("u-hide")
+    showMoreBtn.classList.add("u-hide")
+    showLessBtn.classList.remove("u-hide")
+  } else {
+    detail.classList.add("u-hide")
+  }
+}
+function hideProgressDetail(stage) {
+  const showMoreBtn = document.querySelector(`.show-more-${stage}`);
+  const showLessBtn = document.querySelector(`.show-less-${stage}`);
+  const detail = document.querySelector(`.progress-detail-${stage}`);
+  if (!detail.classList.contains("u-hide")) {
+    detail.classList.add("u-hide")
+    showMoreBtn.classList.remove("u-hide")
+    showLessBtn.classList.add("u-hide")
+  }
+}
+</script>
 {% if application["status"] == "active" %}
-<section class="p-strip--suru-topped u-no-padding--bottom">
-    <div class="row">
-        <h1>Thanks for applying {{ candidate["first_name"] }}!</h1>
-        {% if application['source']['id'] == 2 %}
-            <p>You applied for the role: <a href="/careers/{{ application['job_post_id'] }}"><strong>{{ application["jobs"][0]["name"] }}</strong></a></p>
-        {% else %}
-            <p>You applied for the role: <strong>{{ application["jobs"][0]["name"] }}</strong></p>
+  <section class="p-strip--light is-deep">
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <h1 class="p-heading--2">Thanks for applying, {{ candidate["first_name"] }}.</h1>
+        {% if "stage_progress" in application and application["stage_progress"]["assessment"] and not application["stage_progress"]["early_stage"]  %}
+            {% if application['source']['id'] == 2 %}
+              <p class="u-no-margin--bottom">We have received your application for the role: <a href="/careers/{{ application['job_post_id'] }}">{{ application["jobs"][0]["name"] }}</a></p>
+            {% else %}
+              <p class="u-no-margin--bottom">We have received your application for the role: {{ application["jobs"][0]["name"] }}</p>
+            {% endif %}
+            <p>Your hiring lead is {{ hiring_lead["name"] }}.</p>
+        {% elif "stage_progress" in application and application["stage_progress"]["early_stage"] %}
+            {% if application['source']['id'] == 2 %}
+              <p>We have received your application for the role: <a href="/careers/{{ application['job_post_id'] }}">{{ application["jobs"][0]["name"] }}</a></p>
+            {% else %}
+              <p>We have received your application for the role: {{ application["jobs"][0]["name"] }}</p>
+            {% endif %}
+            {% if candidate["application_ids"]|length >= 2 %}
+              <p class="p-text--x-small-capitalised u-align-text--x-small-to-default u-no-margin--bottom">Status of your other applications:</p>
```

Comment:
The spacing at the bottom still differs from the mockup but I'm not sure if it was decided that it wasn't an issue during design review
![Screenshot 2022-08-23 at 17 44 57](https://user-images.githubusercontent.com/17607612/186202834-f48fdd09-4d6e-4680-bc4d-a7aacb70a0d1.png)

Reply:
![image](https://user-images.githubusercontent.com/57550290/186207526-79f74cf9-a0fc-43f7-88c3-d9e7f3ec92ab.png)

@lyubomir-popov Hi Lyobu! I think these classes have been updated by you the other day. What do you think about the spacing here? :)

Reply:
hi can you please remove `u-no-margin--bottom`. I meant to do thjis myself but didn't get around to doing it, sorry.